### DISCUSSION TITLE
Android Tablet Usage

### DISCUSSION THREAD

**Discussion by noborny** [marcusolsson/obsidian-projects#334] [Help] [answered]
I tend to use Obsidian on my Android Tablet. It's as large as a latop and has a full, physical keyboard but is detected as a mobile device and hence I can't use this plugin. Is there a way for me to 'force' the installation on my tablet? 

Thanks!

**Comment by marcusolsson** [comment, marked as answer]
See [What about mobile support?](https://github.com/marcusolsson/obsidian-projects/discussions/308) under FAQ.

**Reply by cyberyanka** [reply]
I want to express my interest in the mobile version too!
I now work exclusevly on my 14 inch tablet with pysical keyboard and before I switched I made my task tracking to rely heavily on this plugin.

So I'll patiently wait for it and if necessary (and possible) I can be a test subject if you need one!

**Reply by marcusolsson** [reply]
Thanks! The main problem is that to be able to test it, I need to enable it for everyone. Which means it's going to be broken for everyone while I'm working on it. I want to make sure I have dedicated time to work on this before enabling it.

**Reply by adambuchweitz** [reply]
Have you seen https://github.com/TfTHacker/obsidian42-brat? I wonder if it would be helpful?

**Reply by marcusolsson** [reply]
AFAIK BRAT is mainly useful for testing a plugin before it's released. Would be nice if it supported multiple release channels, like alpha, beta, and stable.
